### ISSUE TITLE
Test Step has a labbelled by value

### ISSUE THREAD

**Issue by djbuckley** [manchesergit/material-ui#193]
If Step has a StepLabel or StepButton child it should have a reference to their ID in the returned aria-labelledby tag

this may be covered by accessibility tests

**Comment by davidjacob93** [comment]
Unit tests have been added for the issue. Both of them fail. So added a new issue #237 to the backlog.
